config/locales/en.yml

```diff
@@ -1232,187 +1232,187 @@ en:
+          organic: "Organic Food Store"
+          outdoor: "Outdoor Recreation Store"
+          paint: "Paint Store"
           pastry: "Pastry Shop"
-          pawnbroker: "Pawnbroker"
-          perfumery: "Perfumery"
-          pet: "Pet Shop"
-          pet_grooming: "Pet Grooming"
+          pawnbroker: "Pawnshop"
+          perfumery: "Perfume Store"
+          pet: "Pet Store"
+          pet_grooming: "Pet Groomer"
           photo: "Photo Shop"
-          seafood: "Sea Food"
-          second_hand: "Second-hand Shop"
-          sewing: "Sewing Shop"
-          shoes: "Shoe Shop"
-          sports: "Sports Shop"
-          stationery: "Stationery Shop"
+          seafood: "Seafood Store"
+          second_hand: "Thrift Store"
+          sewing: "Sewing Supply Store"
+          shoes: "Shoe Store"
+          sports: "Sporting Goods Store"
+          stationery: "Office Supply Store"
           storage_rental: "Storage Rental"
           supermarket: "Supermarket"
           tailor: "Tailor"
-          tattoo: "Tattoo Shop"
-          tea: "Tea Shop"
-          ticket: "Ticket Shop"
+          tattoo: "Tattoo Parlor"
+          tea: "Tea Store"
+          ticket: "Ticket Vendor"
           tobacco: "Tobacco Shop"
-          toys: "Toy Shop"
+          toys: "Toystore"
           travel_agency: "Travel Agency"
-          tyres: "Tyre Shop"
-          vacant: "Vacant Shop"
-          variety_store: "Variety Store"
-          video: "Video Shop"
+          tyres: "Tire Store"
+          vacant: "Vacant Storefront"
+          variety_store: "Discount Store"
+          video: "Video Store"
           video_games: "Video Game Store"
           wholesale: "Wholesale Store"
           wine: "Wine Store"
-          "yes": "Shop"
+          "yes": "Store"
         tourism:
-          alpine_hut: "Alpine Hut"
-          apartment: "Holiday Apartment"
-          artwork: "Artwork"
-          attraction: "Attraction"
+          alpine_hut: "Mountain Lodge"
+          apartment: "Guest Apartment"
+          artwork: "Work of Art"
+          attraction: "Tourist Attraction"
           bed_and_breakfast: "Bed and Breakfast"
           cabin: "Tourist Cabin"
-          camp_pitch: "Camp Pitch"
-          camp_site: "Camp Site"
-          caravan_site: "Caravan Site"
-          chalet: "Chalet"
-          gallery: "Gallery"
-          guest_house: "Guest House"
+          camp_pitch: "Campsite"
+          camp_site: "Campground"
+          caravan_site: "RV Park"
+          chalet: "Holiday Home"
+          gallery: "Art Gallery"
+          guest_house: "Guesthouse"
           hostel: "Hostel"
           hotel: "Hotel"
           information: "Information"
           motel: "Motel"
           museum: "Museum"
           picnic_site: "Picnic Site"
-          theme_park: "Theme Park"
-          viewpoint: "Viewpoint"
-          wilderness_hut: "Wilderness Hut"
+          theme_park: "Amusement Park"
+          viewpoint: "Vista Point"
```

Comment:
Is it more common?  I'm pretty familiar with the term "viewpoint" ([example of use by the US National Park Service](https://www.nps.gov/yose/planyourvisit/viewpoints.htm)).  "Vista" or "vista point" seem somewhat familiar, but I wouldn't say more common.

Reply:
Hm, maybe I’m just more familiar with the highway signs that say “vista point” (hardly ever “viewpoint”). I’ll revert it back to “Viewpoint” since at least it isn’t too confusing.